lmdeploy/pytorch/backends/base.py

```diff
@@ -31,6 +31,7 @@ class OpType(Enum):
     FusedMoEW8A8 = auto()
     LinearBlockedF8 = auto()
     FusedMoEBlockedF8 = auto()
+    DeepGemmBlockedF8 = auto()
```

Comment:
this optype is not necessary.
backend op should be device agnostic. And we already dispatch it to `LinearBlockedFP8`